models.py

```diff
@@ -11,10 +10,11 @@ class ToDo(Base):
     id: Mapped[int] = mapped_column(Integer, primary_key=True, index=True)
     content: Mapped[str] = mapped_column(String)
     session_key: Mapped[str] = mapped_column(String)
+    notes: Mapped[str | None] = mapped_column(Text, nullable=True)  # New column added
 
 
-def create_todo(db: Session, content: str, session_key: str):
```

Comment:
Consider using a more specific type for `session_key`, such as `UUID`, if it's meant to be a unique identifier.